charts/traction/values-pr.yaml

```diff
@@ -21,7 +21,7 @@ acapy:
       printKey: true
       printToken: true
       reservationExpiryMinutes: 2880
-      reservationAutoApprove: true
+      reservationAutoApprove: false
```

Comment:
Left this true on a previous PR. Might be useful to have auto approve for pull requests, but also might want to adjust reservation flow and test it in a PR 🤷‍♂️ can alter on a case-by-case basis when needed